ovis_robotiq_gripper/msg/OvisGripper_robot_input.msg

```diff
@@ -0,0 +1,8 @@
+uint8 gACT #getActivation
```

Comment:
Let me know if that's possible or not, could we simply use the commented section at the variable instead of an obscure name? Using `getActivation` instead of `gACT`?